src/main/java/net/laboulangerie/laboulangeriemmo/betonquest/LevelCondition.java

```diff
@@ -5,14 +5,15 @@
 import org.bukkit.Bukkit;
 
 import net.laboulangerie.laboulangeriemmo.LaBoulangerieMmo;
-import net.laboulangerie.laboulangeriemmo.player.MmoPlayer;
+import net.laboulangerie.laboulangeriemmo.api.player.MmoPlayer;
 import pl.betoncraft.betonquest.Instruction;
 import pl.betoncraft.betonquest.api.Condition;
 import pl.betoncraft.betonquest.exceptions.InstructionParseException;
 import pl.betoncraft.betonquest.exceptions.QuestRuntimeException;
 
 public class LevelCondition extends Condition {
 
+    @SuppressWarnings("deprecation")
```

Comment:
Useless no ? Unless `pl.betoncraft.betonquest.Instruction` is deprecated

Reply:
This constructor is deprecated because the next major version of Betonquest won't support it but the replacement constructor isn't implemented yet.